advice.c

```diff
@@ -20,6 +21,33 @@ int advice_add_embedded_repo = 1;
 int advice_ignored_hook = 1;
 int advice_waiting_for_editor = 1;
 
+static int advice_use_color = -1;
```

Comment:
Good, this follows the pattern e.g. in `builtin/branch.c`. However, there is no associated config setting. Maybe handle `advice.color` as a bool in `git_default_advice_config()`?

Reply:
(Similar comments apply to the other `*_use_color` variables below.)